http/codegen/openapi/v3/types.go

```diff
@@ -74,6 +74,12 @@ func buildBodyTypes(api *expr.APIExpr) (map[string]map[string]*EndpointBodies, m
 			}
 
 			req := sf.schemafy(e.Body)
+			if e.SkipRequestBodyEncodeDecode {
+				// it is possible for design to have request Body as Empty and set
+				// SkipRequestBodyEncodeDecode. In this case, we set the request body
+				// example from the Payload.
+				req = sf.schemafy(e.MethodExpr.Payload)
+			}
```

Comment:
Should the code remove any header or param from the payload for the body example?